1-org/envs/shared/variables.tf

```diff
@@ -49,6 +49,18 @@ variable "domains_to_allow" {
   type        = list(string)
 }
 
+variable "enable_shielded_vm_policy" {
```

Comment:
@daniel-cit can we document the caveats for enabling each of these as well (i.e GKE issue with os login and dataflow/composer with Shielded VMs) so that the user is aware.

Reply:
I think its best to link to public docs as @daniel-cit has done, so that they can be the source of truth for any limitations

Reply:
(we should try to get docs updated where they are lacking)